File: Python/replicator_api.py
print("Replycator API 0.01")

import os 

def delete_files():
    for filename in os.listdir("files/"):
        os.remove("files/" + filename)

def send(mode, file_type=None):

    if mode == 'social':

        if file_type == 'all':
            check_send = os.system('sh replycator.sh --social=twitter --text=true --files=true')
        else:
            check_send = os.system('sh replycator.sh --social=twitter --files=true')

        if check_send == 0:
            delete_files()
            return True
        elif check_send == 1:
            return False

    elif mode == 'youtube':
        os.system('sh replycator.sh --social=youtube')

        if check_send == 0:
            delete_files()
            return True
        elif check_send == 1:
            return False

File: C#/replicator_api.cs
using System;
using System.IO;
using System.Diagnostics;

Console.WriteLine("Replycator API 0.01");

void DeleteFiles()
{
    string directoryPath = "files/";
    foreach (string filename in Directory.GetFiles(directoryPath))
    {
        File.Delete(filename);
    }
}

bool Send(string mode, string fileType = null)
{
    if (mode == "social")
    {
        string command;
        if (fileType == "all")
        {
            command = "sh replycator.sh --social=twitter --text=true --files=true";
        }
        else
        {
            command = "sh replycator.sh --social=twitter --files=true";
        }

        int checkSend = RunCommand(command);
        if (checkSend == 0)
        {
            DeleteFiles();
            return true;
        }
        else if (checkSend == 1)
        {
            return false;
        }
    }
    else if (mode == "youtube")
    {
        int checkSend = RunCommand("sh replycator.sh --social=youtube");
        if (checkSend == 0)
        {
            DeleteFiles();
            return true;
        }
        else if (checkSend == 1)
        {
            return false;
        }
    }

    return false;
}

int RunCommand(string command)
{
    ProcessStartInfo startInfo = new ProcessStartInfo
    {
        FileName = "/bin/bash",
        Arguments = "-c \"" + command + "\"",
        RedirectStandardOutput = true,
        RedirectStandardError = true,
        UseShellExecute = false,
        CreateNoWindow = true
    };

    Process process = Process.Start(startInfo);
    process.WaitForExit();
    return process.ExitCode;
}

